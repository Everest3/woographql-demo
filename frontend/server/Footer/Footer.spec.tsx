import React from 'react';
import '@testing-library/react/dont-cleanup-after-each';

import { render, cleanup } from '@axis/testing';
import { Footer } from '.';

describe('DocsNav component', () => {
  afterAll(() => cleanup());

  process.env.GITHUB_URL = 'http://google.com';
  process.env.TWITTER_URL = 'http://google.com';
  process.env.SLACK_URL = 'http://google.com';
  const { baseElement } = render(
    <Footer />,
  );

  it('should render mount successfully', () => {
    expect(baseElement).toMatchSnapshot();
  });
});
