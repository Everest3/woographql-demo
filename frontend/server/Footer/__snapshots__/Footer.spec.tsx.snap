// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocsNav component should render mount successfully 1`] = `
<body>
  <div>
    <div
      class="w-full mx-auto bg-cello text-white h-48"
    >
      <div
        class="min-h-full h-full w-full mx-auto max-w-screen-lg flex flex-wrap  justify-center pt-3 px-5 lg:px-6 lg:flex-nowrap lg:items-center"
      >
        <small
          class="basis-auto mr-3"
        >
          ©  2021-
          2023
        </small>
        <small
          class="basis-auto flex-shrink lg:mr-3"
        >
          <strong>
            WPGraphQL WooCommerce
          </strong>
          .
        </small>
        <small
          class="basis-auto mb-6 lg:mb-0"
        >
          All rights reserved.
        </small>
        <div
          class="flex order-first justify-center lg:order-last lg:justify-end items-center basis-full lg:basis-1/3"
        >
          <a
            class="icon-button text-2xl h-16 pr-5 text-white"
            href="http://google.com"
          >
            <i
              class="fa-brands fa-github"
            />
          </a>
          <a
            class="icon-button text-2xl h-16 pr-5 text-white"
            href="http://google.com"
          >
            <i
              class="fa-brands fa-twitter"
            />
          </a>
          <a
            class="icon-button text-2xl h-16  text-white"
            href="http://google.com"
          >
            <i
              class="fa-brands fa-slack"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</body>
`;
